sql/server_component/log_sink_trad.cc

```diff
@@ -349,8 +350,23 @@ int log_sink_trad(void *instance MY_ATTRIBUTE((unused)), log_line *ll) {
         log_sink_trad_last = microtime;
       }
 
+      bool log_to_buffered_error_log = false;
+      bool log_only_to_buffered_error_log = false;
+
+      // Add product specific buffered_error_log logic based on subsys & prio
+      // here
```

Comment:
Should we add the control flag elsewhere?
LIke this?: LogEvent().module("blah").buffered(true)

Reply:
I'm not sure about this, as the idea is to log everything printed by a submodule or loglevel to this log. That condition is easier to implement directly in log_sink_trad instead of adding it to all the macros/places log messages are generated.

Reply:
ok, lets leave it as is